app/src/main/java/com/x8bit/bitwarden/ui/platform/feature/vaultunlockednavbar/VaultUnlockedNavBarScreen.kt

```diff
@@ -277,18 +280,30 @@ private fun VaultBottomAppBar(
 
             NavigationBarItem(
                 icon = {
-                    Icon(
-                        painter = rememberVectorPainter(
-                            id = if (isSelected) {
-                                destination.iconResSelected
-                            } else {
-                                destination.iconRes
-                            },
-                        ),
-                        contentDescription = stringResource(
-                            id = destination.contentDescriptionRes,
-                        ),
-                    )
+                    BadgedBox(
+                        badge = {
+                            this@BottomAppBar.AnimatedVisibility(
+                                visible = destination.notificationCount > 0,
```

Comment:
How does the `AnimatedVisibility` look here?

Mostly curious how using the `RowScope.AnimatedVisibility` interacts with this `BoxScope`.

Reply:
Set up some code to remove the badge and this is the results, I did update the transitions as the default looked a bit janky for the row scope one ->

| explicit compose.animiation | rowScoped |
|--------|--------|
| https://github.com/user-attachments/assets/a4294982-c3d7-4128-9a08-da2cdb3a33e3 | https://github.com/user-attachments/assets/dc5206a6-1d09-4bfa-bd2f-683e05f3a675 |

Reply:
basically can call it like `androidx.compose.animation.AnimatedVisibility()` here or leave it as it is.

Reply:
I am going to lean toward the explicit call and place a comment just cause the crossing the Scope is kinda funky.